src/platform/packages/shared/kbn-scout/src/cli/config_discovery.ts

```diff
@@ -21,29 +22,38 @@ export const discoverPlaywrightConfigs: Command<void> = {
 
   Common usage:
     node scripts/scout discover-playwright-configs --searchPaths <search_paths>
+    node scripts/scout discover-playwright-configs --save
     node scripts/scout discover-playwright-configs
   `,
   flags: {
     string: ['searchPaths'],
-    default: { searchPaths: DEFAULT_TEST_PATH_PATTERNS },
+    boolean: ['save'],
+    default: { searchPaths: DEFAULT_TEST_PATH_PATTERNS, save: false },
   },
   run: ({ flagsReader, log }) => {
     const searchPaths = flagsReader.arrayOfStrings('searchPaths')!;
 
-    const plugins = measurePerformance(log, 'Discovering playwright config files', () => {
+    const pluginsMap = measurePerformance(log, 'Discovering playwright config files', () => {
       return getScoutPlaywrightConfigs(searchPaths, log);
     });
 
     const finalMessage =
-      plugins.size === 0
-        ? 'No playwright config files found'
-        : `Found playwright config files in '${plugins.size}' plugins`;
+      pluginsMap.size === 0
+        ? 'No Playwright config files found'
+        : `Found Playwright config files in '${pluginsMap.size}' plugins`;
+
+    if (pluginsMap.size > 0 && flagsReader.boolean('save')) {
+      const filename = 'scout_test_configs.json';
```

Comment:
I think it would be good to save that generated file under `REPO_ROOT/.scout` such that it's also covered by our gitignore rule. What do you think?

Reply:
good point, changing it

Reply:
bfa4b763a9e576559a999f0103f0fe643668fc14